stats/cloud/data.go

```diff
@@ -25,47 +25,28 @@ import (
 	"fmt"
 	"math"
 	"sort"
-	"strconv"
 	"time"
 
 	"github.com/loadimpact/k6/lib/metrics"
 	"github.com/loadimpact/k6/lib/netext/httpext"
 	"github.com/loadimpact/k6/stats"
 )
 
-const DataTypeSingle = "Point"
-const DataTypeMap = "Points"
-const DataTypeAggregatedHTTPReqs = "AggregatedPoints"
-
-// Timestamp is used for sending times encoded as microsecond UNIX timestamps to the cloud servers
-type Timestamp time.Time
+//noling:golint
```

Comment:
The typo in `noling` is probably the only reason `nolintlint` hasn't complained

Reply:
:rofl: 
if I remove it ... I get `golint` complaining about lack of comments ... if I fix the typo ... I get `nolintlint` complaining  :exploding_head:

Reply:
wtf :rofl:

Reply:
adding a comment fixed it .. so I will make fix it that way